src/js/locale/pl.yaml

```diff
@@ -0,0 +1,532 @@
+    popularity: Popularność
+time:
+  seconds: 
+    short: sek
+    singular: sekunda
+    plural: sekund
+  minutes: 
+    short: min
+    singular: minuta
+    plural: minut
+  hours: 
+    short: gdz
+    singular: godzina
+    plural: godzin
+  days: 
+    short: d
+    singular: dzień
+    plural: dni
+  weeks: 
+    short: tyg
+    singular: tydzień
+    plural: tygodni
+  years: 
+    short: r
+    singular: rok
+    plural: lat  # i love this inconsistency
+specs:
+  tracks: '%{count} ścieżek'
+  albums: '%{count} albumów'
+  followers: '%{count} obserwujących'
+  listeners: '%{count} słuchających'
+  playlists: '%{count} playlist'
+  plays: '%{count} odtworzeń'
+  popularity: '%{percent}% popularności'
+  added: 'Dodano '
+  added_ago: 'Dodano %{time} temu'
+  edited: 'Edytowano '
+  edited_ago: 'Edytowano %{time} temu'
+  played_ago: 'Odtworzono %{time} temu'
+services:
+  mopidy:
+    title: Mopidy
+    local: Lokalnie
+    adding_uris: 'Dodawanie %{count} URI'
+    searching_providers: 'Wyszukiwanie %{count} dostawców Mopidy'
+    loading_albums: 'Ładowanie %{count} lokalnych albumów'
+  spotify:
+    title: Spotify
+  pusher:
+    title: Pusher
+  snapcast:
+    title: Snapcast
+  genius:
+    title: Genius
+    switch_lyrics_result: Przełącz na inne wyniki wyszukiwania tekstu utworu
+    want_lyrics: 'Potrzebujesz tekstu utworu? Autoryzuj Geniusa w '
+  google:
+    title: Google
+  icecast:
+    title: Icecast
+  lastfm:
+    title: LastFM
+    love: Lubię to
+    unlove: Nie lubię tego
+
+##
+# Components
+##
+sidebar:
+  now_playing: Teraz odtwarzane
+  search: Szukaj
+  discover: Odkrywaj
+  genre: Gatunek / Nastrój
+  featured_playlists: Proponowane playlisty
+  new_releases: Nowości
+  my_music: Moja muzyka
+  playlists: Playlisty
+  artists: Wykonawcy
+  albums: Albumy
+  tracks: Utwory
+  browse: Przeglądaj
+  settings: Ustawienia
+  not_connected: '%{name} nie połączony'
+  browser_offline: Przeglądarka offline
+  update_available: Aktualizacja dostępna
+context_menu:
+  add_to_playlist:
+    title: Dodaj do playlisty
+    no_playlists: Brak edytowalnych playlist
+  play_next: Odtwarzaj następne
+  play_top_tracks: Odtwarzaj najlepsze
+  love_track: Lubię to
+  unlove_track: Nie lubię tego
+  pin: Przypnij
+  unpin: Odepnij
+  go_to_artist: Przejdź do wykonawcy
+  go_to_user: Przejdź do użytkownika
+  track_info: Informacje o utworze
+  discover_similar: Odkryj podobne
+  start_radio: Włącz radio
+  copy_uri: Kopiuj URI
+dropzones:
+  dragging_things: 'Przeciąganie %{count} obiektów'
+playback_controls:
+  consume: Konsumuj
+  shuffle: Losuj
+  repeat: Powtarzaj
+  no_outputs: Brak wyjść
+  mute: Wycisz
+  unmute: Wyłącz wyciszenie
+snapcast:
+  enabled: Odblokowane
+  show_disconnected_clients: Pokaż rozłączone klienty
+  no_connected_clients: Brak połączonych klientów
+  host: Host
+  port: Port
+  name: Nazwa
+  name_disconnected: Nazwa (rozłączono)
+  group: Grupa
+  new_group: '+ Nowa grupa'
+  latency: Latencja

```

Comment:
`Opóźnienie` sounds better.

Reply:
In technical lanugage `Latencja` is imo more suitable

Reply:
Which is a literal translation making no sense. Why would you assume the language needs to be technical? This software is for everyone.
> latencja «okres utajnienia choroby, popędu seksualnego u dzieci itp.»

Reply:
https://zakamarkiaudio.pl/2013/01/latencja-i-sposoby-jej-minimalizowania.html
https://nagrywamy.com/2020/05/06/wylaczanie-kompensacji-latencji-w-reaperze-sposob-drugi-oczywisty/
https://muzyczny.pl/portal/latencja-czyli-co-to-jest-i-jak-sobie-z-tym-radzic/
And most importantly: https://pl.wiktionary.org/wiki/latencja

This term is actually being used very commonly in that meaning and context. Also, when I hear "latencja" I think of a more or less regular delay of a streamed signal which matches this case very well. The difference is not deal changing tho, but leaves less doubts what is it about (i.e, something different than lag of regular requests).

Reply:
Fair enough.